assets/src/variations/stretched-link/index.scss

```diff
@@ -4,9 +4,10 @@
   [class*="is-style-rounded-"].force-no-caption {
     overflow: hidden;
     border-radius: 50%;
```

Comment:
Not part of the scope, but do we also need this property if it's already defined [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/core-overrides/Image.scss#L4)?

Reply:
Yes, this one is for the parent (the element with `is-style-rounded` class) whereas the other one is for the child (either an `img` or a `figure` element)
We need both for the animation on hover, otherwise it looks off:
<img width="312" alt="Screenshot 2023-02-28 at 15 20 14" src="https://user-images.githubusercontent.com/6949075/221881229-0ba0fae4-56ad-480d-afe3-ef0c38b77415.png">